### PR TITLE
Support for YouTube media tags.

### PR DESCRIPTION
**Pull request by PitchRE**
Hello. I added support for YouTube media tags. 

image

image

### CODE REVIEW

File: src/Entity/Video.php [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?php
 
-declare(strict_types=1);
+declare (strict_types = 1);
```
Comment (norkunas) [inline src/Entity/Video.php:3]: Please revert :)

Comment (PitchRE) [reply src/Entity/Video.php]: commit 455c3cb26c2e1f951e9659c44c90b70f065d1a7c

Comment (PitchRE) [reply src/Entity/Video.php]: ehh. My IDE changed it again. commit 10bf12d47c63ca6116839fcc06cce1516c1925b7 should be ok.

File: src/Entity/Video.php [hunk 2]
```diff
@@ -15,6 +15,30 @@ class Video extends AbstractEntity
...
+    }
+
+    /**
+     * @return string
+     */
+    public function getTrack()
+    {
+        return $this->get('track');
+    }
+
```
Comment (norkunas) [inline src/Entity/Video.php:41]: Are these columns always set? For example if using another provider than youtube? Could they return null?

Comment (PitchRE) [reply src/Entity/Video.php:41]: Youtube media tags are not always available, so yeah - they can return null. 

> The description of a video that contains music may include information about the song(s) in the video. YouTube adds this information automatically to some videos when the video has been identified and “claimed” by music rightsholders  via YouTube’s rights management tools, including Content ID.

Comment (norkunas) [reply src/Entity/Video.php:41]: Could you update return typehints then? :)

Comment (PitchRE) [reply src/Entity/Video.php]: sure. commit 860bc57d14ad0f68b23e3b478804011a40441766

**Comment by norkunas** [comment]
Also forgot to say - thanks :)

**Pull request merged**